Branch-SDK/src/io/branch/referral/SystemObserver.java

```diff
@@ -324,7 +324,8 @@ public boolean getBluetoothPresent() {
         try {
             BluetoothAdapter bluetoothAdapter = BluetoothAdapter.getDefaultAdapter();
             if (bluetoothAdapter != null) {
-                return bluetoothAdapter.isEnabled();
+				//noinspection ResourceType
```

Comment:
Hmm, I'm not seeing this method locally. Are you seeing it when you include in your project (and perhaps don't have Bluetooth enabled)?

Should we allow that? I thought we required Bluetooth to be enabled.